docs/user_guide/quickstart.rst

```diff
@@ -133,6 +128,23 @@ are placed in the ``build/<design>`` directory.
    tasktime            0.35         1.4           0.83          0.6           0.8           1.71          2.69          0.66          2.2          0.74
 
 
+If you find that having all the tool-specific information printed to the screen is too noisy, you can set the :keypath:`option,quiet` parameter to True.
+
+.. code-block:: bash
+
+   chip.set('option','quiet',True)
+   
+
+Even if you use the :ref:`quiet option <quiet>`, you will still be able to access the tool-specific output from the log file that is saved for each tool. You can find your log files by running:
+
+.. code-block:: bash
+
+   find build/<design>/<jobname>/* -name "\*.log\*"
```

Comment:
I think this might be a little too in the weeds.
begin specific about it might be okay to say: build/design/jobname/step/index/step.log